core/storage/suggestion/gae_models.py

```diff
@@ -1990,3 +2000,1275 @@ def export_data(
+            'accepted_translation_word_count':
+                base_models.EXPORT_POLICY.EXPORTED,
+            'rejected_translations_count':
+                base_models.EXPORT_POLICY.EXPORTED,
+            'first_contribution_date':
+                base_models.EXPORT_POLICY.EXPORTED,
+            'last_contribution_date':
+                base_models.EXPORT_POLICY.EXPORTED
+        })
+
+    @classmethod
+    def apply_deletion_policy(cls, user_id: str) -> None:
+        """Delete instances of TranslationReviewerTotalContributionStatsModel
+        for the user.
+
+        Args:
+            user_id: str. The ID of the user whose data should be deleted.
+        """
+        datastore_services.delete_multi(
+            cls.query(cls.contributor_id == user_id).fetch(keys_only=True))
+
+    @classmethod
+    def export_data(
+        cls, user_id: str
+    ) -> Dict[str, Dict[str, Union[str, int, List[str]]]]:
+        """Exports the data from TranslationReviewerTotalContributionStatsModel
+        into dict format for Takeout.
+
+        Args:
+            user_id: str. The ID of the user whose data should be exported.
+
+        Returns:
+            dict. Dictionary of the data from
+            TranslationReviewerTotalContributionStatsModel.
+        """
+        user_data = {}
+        stats_models: Sequence[TranslationReviewerTotalContributionStatsModel] = ( # pylint: disable=line-too-long
+            cls.get_all().filter(cls.contributor_id == user_id).fetch())
+        for model in stats_models:
+            splitted_id = model.id.split('.')
+            id_without_user_id = splitted_id[0]
+            user_data[id_without_user_id] = {
+                'language_code': model.language_code,
+                'topic_ids_with_translation_reviews': (
+                    model.topic_ids_with_translation_reviews),
+                'reviewed_translations_count': (
+                    model.reviewed_translations_count),
+                'accepted_translations_count': (
+                    model.accepted_translations_count),
+                'accepted_translations_with_reviewer_edits_count': (
+                    model.accepted_translations_with_reviewer_edits_count),
+                'accepted_translation_word_count': (
+                    model.accepted_translation_word_count),
+                'rejected_translations_count': (
+                    model.rejected_translations_count),
+                'first_contribution_date': (
+                    model.first_contribution_date.isoformat()),
+                'last_contribution_date': (
+                    model.last_contribution_date.isoformat())
+            }
+        return user_data
+
+
+class QuestionSubmitterTotalContributionStatsModel(base_models.BaseModel):
+    """Records the question contribution stats for contributor dashboard admin
+    page. There is one instance of this model per contributor_id.
+    Its IDs is same as [contributor_id].
```

Comment:
`The ID of each model is simply: [contributor_id].`

(There is only one ID for each model, so "IDs" is misleading.)

Reply:
Okay. Changed.